lib/libxdp/README.org

```diff
@@ -212,9 +215,165 @@ follows:
 If set, the =LIBXDP_BPFFS= environment variable will override the location of
 =bpffs=, but the =xdp= subdirectory is always used.
 
+* Using AF_XDP sockets
+
+Libxdp implements helper functions for configuring AF_XDP sockets as
+well as reading and writing packets from these sockets. AF_XDP sockets
+can be used to redirect packets to user-space at high throughputs from
+an XDP program. Note that this functionality used to reside in libbpf,
+but has now been moved over to libxdp as it is a better fit for this
+library. As of the 1.0 release of libbpf, the AF_XDP socket support
+will (or has) be removed and all future development will be performed
```

Comment:
maybe just drop the (or has) for now, and we can change it once libbpf actually removes the functionality?